src/main/java/software/amazon/cloudformation/exceptions/CfnInvalidCredentialsException.java

```diff
@@ -22,10 +22,10 @@
     private static final HandlerErrorCode ERROR_CODE = HandlerErrorCode.InvalidCredentials;
 
     public CfnInvalidCredentialsException() {
-        this(null);
+        super(ERROR_CODE.getMessage(), ERROR_CODE);
     }
 
     public CfnInvalidCredentialsException(final Throwable cause) {
```

Comment:
I don't see any reason to limit all of these exceptions to just having a `cause` constructor.  Can we include a `(cause)` and a `(message, cause)` for all of them?

Reply:
can we create an issue to resolve this then, if we're not going to include it in this PR?

Reply:
Tracked in #276